app/src/ui/about/about.tsx

```diff
@@ -169,11 +171,13 @@ export class About extends React.Component<IAboutProps, IAboutState> {
           : ''
 
         return (
-          <Row>
-            <Button disabled={disabled} onClick={onClick} tooltip={tooltip}>
-              {buttonTitle}
-            </Button>
-          </Row>
+          <>
+            <Row>
+              <Button disabled={disabled} onClick={onClick} tooltip={tooltip}>
+                {buttonTitle}
+              </Button>
+            </Row>
+          </>
```

Comment:
Why the `<>...</>` ? 🤔 
```suggestion
          <Row>
            <Button disabled={disabled} onClick={onClick} tooltip={tooltip}>
              {buttonTitle}
            </Button>
          </Row>
```